base/asyncmap.jl

```diff
@@ -62,7 +62,6 @@ function done(itr::AsyncCollector, state::AsyncCollectorState)
 end
 
 function next(itr::AsyncCollector, state::AsyncCollectorState)
-
```

Comment:
Noted.

Being consistent with the rest of the codebase here. While not an explicitly stated guideline, the bulk of the existing Julia codebase avoids starting a function definition with a newline.